src/python/bot/tasks/analyze_task.py

```diff
@@ -302,6 +303,11 @@ def execute_task(testcase_id, job_type):
   testcase.crash_type = state.crash_type
   testcase.crash_address = state.crash_address
   testcase.crash_state = state.crash_state
+  crash_stacktrace_output = utils.get_crash_stacktrace_output(
+      application_command_line, state.crash_stacktrace,
+      unsymbolized_crash_stacktrace)
+  testcase.crash_stacktrace = data_handler.filter_stacktrace(
+      crash_stacktrace_output)
```

Comment:
So, we want to store stack for this return case ??

```
  # See if we have to ignore this crash.
  if crash_analyzer.ignore_stacktrace(state.crash_stacktrace):
    close_invalid_testcase_and_update_status(testcase, metadata, 'Irrelavant')
    return
```

Reply:
It seems like we're hanging somewhere below that return, but I did want to handle this case as well. Either way, we can compute everything at the new spot so no reason not to store it as early as possible.